Readme.md

```diff
@@ -366,6 +316,12 @@ class Organization
      */
     public $projects = [];
 
+    public function __construct(string $identifier, string $name)
+    {
+        $this->identifier = $identifier;
+        $this->name = $name;
+    }
```

Comment:
Not changed here, but maybe we should add a comment why the `@Flow\Entity` annotation is used at the class declaration